### ISSUE TITLE
element.NotifyAriaStateChange(string) and related example is confusing

### ISSUE THREAD

**Issue by domenic** [cyns/wapa#6]
This is not described in enough detail to know what it's doing. And the example under "Accessibility information Stored outside DOM" is pretty confusing; it's actually broken in that the ToggleMe function is not exposed to the global scope and so cannot be called.

Does this literally set the attribute, like the text implies? If so it seems not very useful. Does it cause commands to fire somehow? If so, why---what is the processing model?

In general, how is that example supposed to work? Is the checkbox supposed to work with enter/spacebar/etc., or just clicking? Is it supposed to be focusable? Etc.
